client/my-sites/marketing/do-it-for-me/difm-landing.tsx

```diff
@@ -248,15 +248,24 @@ export default function DIFMLanding( {
 	const dispatch = useDispatch();
 
 	const productCost = useSelector( ( state ) => getProductCost( state, WPCOM_DIFM_LITE ) );
+	const tiers = useSelector( ( state ) => getProductPriceTierList( state, WPCOM_DIFM_LITE ) );
+
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
-	const hasPriceDataLoaded = productCost && currencyCode;
+	const hasPriceDataLoaded = productCost && tiers.length > 1 && currencyCode;
 
 	const [ isLoading, setIsLoading ] = useState( true );
 
 	const displayCost = hasPriceDataLoaded
 		? formatCurrency( productCost, currencyCode, { stripZeros: true } )
 		: '';
 
+	const extraPageCost = hasPriceDataLoaded
+		? formatCurrency( tiers[ 1 ].minimum_price - tiers[ 0 ].minimum_price, currencyCode, {
```

Comment:
Could we maybe move this to a function inside the following. 

https://github.com/Automattic/wp-calypso/blob/b12a4deb0e7c39e47a1c017767ee5987db1d01d0/packages/calypso-products/src/get-difm-tiered-price-details.ts#L1-L16

Reply:
Nice catch! I missed this!

Reply:
Hi @jdc91, I have refactored the code so that the logic for parsing the price tiers remains inside this function only. I also have added tests for this function. 🎉 

Can you please have a look?

<img width="686" alt="image" src="https://user-images.githubusercontent.com/5436027/192991828-8ff68678-9ad8-4f58-a76b-7fdc37c25adf.png">